options.html

```diff
@@ -30,11 +30,25 @@
 							<br />
 							<input type="checkbox" id="minihelp" name="minihelp">
 							Show mini help
+							<br />
+							<br />
+							Show popup with delay: 
+							<input type="number" id="popupDelay" name="popupDelay" min="1"> milliseconds
 							</p>
 					</div>
 					<!-- <div id="keycon" class="tabcontent hide"> -->
 					<div id="keycon" class="tabContent">
 					<h1>Keyboard</h1>
+						<input type="checkbox" id="showOnKeyFlag" name="showOnKeyFlag" disabled>
+						Immediately show only when key <select id="showOnKey" name="showOnKey"> 

```

Comment:
This isn't just 'Immediately' right?  Isn't the goal to show definitions *only* when the key is pressed?